src/diff/children.js

```diff
@@ -144,6 +146,7 @@ export function diffChildren(
 
 						outer: if (oldDom == null || oldDom.parentNode !== parentDom) {
 							parentDom.appendChild(newDom);
+							nextDom = null;
```

Comment:
Since we are adding `newDom` to the end of `parentDom`, `newDom.nextSibling` will always be `null`. So instead of going through the DOM APIs to get this, I'm precalculating it here so save us a DOM read.